validation/mesoscale_turbulence/baroclinic_adjustment.jl

```diff
@@ -8,7 +8,7 @@ using JLD2
 using Oceananigans
 using Oceananigans.Units
 using Oceananigans.Models.HydrostaticFreeSurfaceModels: fields
-using Oceananigans.TurbulenceClosures: VerticallyImplicit, Vertical, Horizontal
+using Oceananigans.TurbulenceClosures
```

Comment:
Can we delete this line?

Reply:
I explicitly exported 
```
    ExplicitTimeDiscretization,
    VerticallyImplicitTimeDiscretization,
    ThreeDimensionalFormulation,
    HoriozontalFormulation,
    VerticalFormulation
```

in the module `TurbulenceClosures`